website/server/models/group.js

```diff
@@ -370,11 +370,23 @@ export function chatDefaults (msg, user) {
     message.uuid = 'system';
   }
 
+  if (participants) {
+    let infoStr = 'participants: ';
+    for (let i = 0; i < participants.length; i++) {
+      if (i === participants.length - 1) {
+        infoStr += `${participants[i]}`;
+      } else {
+        infoStr += `${participants[i]} ,`;
+      }
+    }
+    message._info = infoStr;
+  }
+
   return message;
 }
 
-schema.methods.sendChat = function sendChat (message, user) {
-  let newMessage = chatDefaults(message, user);
+schema.methods.sendChat = function sendChat (message, user, participants) {
+  let newMessage = chatDefaults(message, user, participants);
```

Comment:
I'd prefer to see something like:

```js
schema.methods.sendChat = function sendChat (message, user, metaData) {
let newMessage = chatDefaults(message, user)
newMessage._meta = metaData;
```

An underscored property, to designate that it is private and should not be relied on (as it could change at any time). Also, an object so that it's extensible and more properties can be added to it if needed.

Then when it is used in the quest start method:

```js
this.sendChat(`Your quest, ${quest.text('en')}, has started.`, null, {
  participatingMembers: this.getParticipatingQuestMembers().join(', '),
});
```

@alys, thoughts?